content/applications/general/developer_mode.rst

```diff
@@ -1,16 +1,13 @@
-==============
-Developer Mode
-==============
-
 .. _developer-mode:
 
-Activate the developer mode (debug mode)
```

Comment:
Without the "Activate", the headings' titles immediately make a lot less sense: 
![image](https://user-images.githubusercontent.com/15896813/124754559-abddea80-df2a-11eb-9b9d-8f07f966bd14.png)